relsext.rdf

```diff
@@ -88,6 +98,38 @@
     <rdfs:label xml:lang="en">generate ocr</rdfs:label>
     <rdfs:range rdf:resource="&xsd;boolean"/>
   </rdf:Property>
+
+  <!-- http://islandora.ca/ontology/relsext#email-admin-when-ingested -->
+  <rdf:Property rdf:about="http://islandora.ca/ontology/relsext#email-admin-when-ingested">
+    <rdfs:label xml:lang="en">email admin when ingested</rdfs:label>
+    <rdfs:range rdf:resource="&xsd;boolean"/>
+  </rdf:Property>
+
+  <!-- http://islandora.ca/ontology/relsext#book-batched -->
+  <rdf:Property rdf:about="http://islandora.ca/ontology/relsext#book-batched">
+    <rdfs:label xml:lang="en">book batched</rdfs:label>
+    <rdfs:range rdf:resource="&xsd;boolean"/>
+  </rdf:Property>
+
+  <!-- http://islandora.ca/ontology/relsext#newspaper-batched -->
+  <rdf:Property rdf:about="http://islandora.ca/ontology/relsext#newspaper-batched">
+    <rdfs:label xml:lang="en">newspaper-batched</rdfs:label>
+    <rdfs:range rdf:resource="&xsd;boolean"/>
+  </rdf:Property>
+  
+  <!-- http://islandora.ca/ontology/relsext#width -->
+  <rdf:Property rdf:about="http://islandora.ca/ontology/relsext#width">
+    <rdfs:label xml:lang="en">width</rdfs:label>
+    <rdfs:comment xml:lang="en">The width of an image</rdfs:comment>
+    <rdfs:range rdf:resource="&xsd;nonNegativeInteger"/>
+    <skos:exactMatch rdf:resource="http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#width" />
+  </rdf:Property>
```

Comment:
There is also a **height** element used, so you could probably add that as well.